src/frontend/apps/desk/src/features/footer/Footer.tsx

```diff
@@ -0,0 +1,172 @@
+  position: absolute;
+  height: 2px;
+  width: 100%;
+  background: var(--c--theme--colors--primary-600);
+  top: 0;
+`;
+
+export const Footer = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Box $position="relative" as="footer">
+      <BlueStripe />
+      <Box $padding={{ top: 'large', horizontal: 'big', bottom: 'small' }}>
+        <Box>
+          <Image
+            priority
+            src={IconMarianne}
+            alt={t('Marianne Logo')}
+            width={70}
+          />
+        </Box>
+        <Box
+          $direction="row"
+          $gap="1.5rem"
+          $align="center"
+          $justify="space-between"
+          $css="flex-wrap: wrap;"
+        >
+          <Box>
+            <Box $align="center" $gap="6rem" $direction="row">
+              <Image
+                width={100}
+                priority
+                src={IconGouv}
+                alt={t('Freedom Equality Fraternity Logo')}
+              />
+            </Box>
+          </Box>
+          <Box
+            $direction="row"
+            $css={`
+              column-gap: 1.5rem;
+              row-gap: .5rem;
+              flex-wrap: wrap;
+            `}
+          >
+            {[
+              {
+                label: 'legifrance.gouv.fr',
+                href: 'https://legifrance.gouv.fr/',
+              },
+              {
+                label: 'info.gouv.fr',
+                href: 'https://info.gouv.fr/',
+              },
+              {
+                label: 'service-public.fr',
+                href: 'https://service-public.fr/',
+              },
+              {
+                label: 'data.gouv.fr',
+                href: 'https://data.gouv.fr/',
+              },
+            ].map(({ label, href }) => (
+              <StyledLink
+                key={label}
+                href={href}
+                target="__blank"
+                $css={`
+                  gap:0.2rem;
+                  transition: box-shadow 0.3s;
+                  &:hover {
+                    box-shadow: 0px 2px 0 0 var(--c--theme--colors--greyscale-text);
+                  }
+                `}
+              >
+                <Text $weight="bold">{label}</Text>
+                <IconLink width={18} />
+              </StyledLink>
+            ))}
+          </Box>
+        </Box>
+        <Box
+          $direction="row"
+          $margin={{ top: 'big' }}
+          $padding={{ top: 'tiny' }}
+          $css="
+            flex-wrap: wrap;
+            border-top: 1px solid var(--c--theme--colors--greyscale-200); 
+            column-gap: 1rem;
+            row-gap: .5rem;
+          "
+        >
+          {[
+            {
+              label: t('Legal Notice'),
+              href: '/legal-notice',
+            },
+            {
+              label: t('Personal data and cookies'),
+              href: '/personal-data-cookies',
+            },
+            {
+              label: t('Accessibility'),
```

Comment:
I added a comment about this label lower in a [footer.spec.ts](https://github.com/numerique-gouv/people/pull/207/files?short_path=5c483ea#diff-dc677f90201539be13c37f322318df9630c536d097f4e01a34bd9ab15f0d9a42) comment

Reply:
See: https://github.com/numerique-gouv/people/pull/207#discussion_r1654712527